tools/triage_status_report.py

```diff
@@ -1,48 +1,78 @@
+#!/usr/bin/env python3
 import json
-import requests
 import argparse
+import dataclasses
+from typing import List
 
 import jira
+import requests
 
 JIRA_SERVER = "https://issues.redhat.com"
 FILTER_ID = 12380672
 
 
-def main(jira_client, filter_id, webhook):
-    filter = jira_client.filter(filter_id)
-    issues = jira_client.search_issues(filter.jql)
-    issues_count = len(issues)
-    if issues_count == 0:
-        return
-
-    filter_url = filter.viewUrl
-    text = f"There are <{filter_url}|{issues_count} new triage tickets>\n"
-    email_domains = list()
-    feature = list()
-    ticket_text = ""
+@dataclasses.dataclass
+class IssueData:
+    key: str
+    url: str
+    user: str
+    email_domain: str
+    features: List[str]
+
+
+def _get_issues_data(issues):
     for issue in issues:
         key = issue.key
         url = issue.permalink()
-        labels = list()
-        for label in issue.raw["fields"]["labels"]:
-            if "FEATURE" in label:
-                labels.append(label.replace("FEATURE-", ""))
-
-        email_domain = issue.raw["fields"]["customfield_12319045"]
         user = issue.raw["fields"]["customfield_12319044"]
-        ticket_text += f"<{url}|{key}>   {user:<15} {email_domain:<15} {labels}\n"
+        email_domain = issue.raw["fields"]["customfield_12319045"]
+
+        features = [label.replace("FEATURE-", "")
+                    for label in issue.raw["fields"]["labels"]
+                    if "FEATURE" in label]
+
+        # filtering some non-important "features"
+        features = [feature for feature in features
+                    if feature not in (
+                        "Requested-hostname",
+                        "Requsted-hostname",  # leaving the value with the typo for backwards compatibility
+                        "NetworkType",
+                    )]
```

Comment:
This should be changed in the service code as well. Open a ticket and add a TODO to change it once the service is updated

Reply:
Already done, already merged
Backwards compatibility baby :)